src/ClusterManager/job_launcher.py

```diff
@@ -523,6 +524,17 @@ def submit_job_impl(self, job):
             job_object.params["uid"] = user_info["uid"]
             job_object.params["user"] = job_object.get_alias()
 
+            if "job_token" not in job_object.params:
+                if "user_sign_token" in config and "userName" in job_object.params:
```

Comment:
Just want to make sure that this is an email, not just an alias.

Reply:
it is email in the current implementation. maybe we need to force the format in portal or restfulapi